.atlascli.goreleaser.yml

```diff
@@ -0,0 +1,81 @@
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - <<: &build_defaults
+      env:
+        - CGO_ENABLED=0
+      binary: bin/atlas
+      main: ./cmd/atlas/atlas.go
+      ldflags:
+        - -s -w -X github.com/mongodb/mongocli/internal/version.Version={{.Version}} -X github.com/mongodb/mongocli/internal/version.GitCommit={{.FullCommit}}
```

Comment:
don't you need the tool name here?

Reply:
We are waiting for Cailin to purchase the license. I have filed https://jira.mongodb.org/browse/CLOUDP-113305 to make this change once we have the license